src/contrib/mlir/core/pass/mlir_subgraph_extraction.cpp

```diff
@@ -418,6 +418,50 @@ void MLIRSubgraphExtractionPass::sanity_check(std::shared_ptr<Function> func, No
     }
 }
 
+template <typename T>
+static bool can_use_mkldnn_conv_callback(ngraph::Node* node)
+{
+    auto convolution = static_cast<const T*>(node);
```

Comment:
Is this method specifically for convolution nodes ? Why templatized when it seems very type-specific ? Maybe specialize the template instead ?

Reply:
This is for the family of convolution nodes: Convolution, ConvolutionBias, ConvolutionRelu, GroupConvolution, ...

Reply:
I see. Then maybe add a comment to that effect.